Jenkinsfile

```diff
@@ -45,6 +45,9 @@ node ('docker') {
         // Build an image from the the local Dockerfile
         def athImg = docker.build('blueocean-ath-builder')
 
+        // fetch Maven configuration
+        configFileProvider([configFile(fileId: 'blueocean-maven-settings', targetLocation: 'settings.xml')]) {
```

Comment:
Hey Yoann ... can we add a comment here wrt what `configFileProvider` and `configFile` are and how they are being picked up by the build? I have no idea what they are anyway and I'm sure others are the same.